### ISSUE TITLE
Vox limb dropping upon gibbing should be removed until newmed

### ISSUE THREAD

**Issue by bruhmogus** [space-wizards/space-station-14#32132]
## Description
Gibbing and explode smite make vox drop limbs. This shouldn't happen to be accurate with other species

**Reproduction**
Step 1. Blow a vox the fuck up
Step 2. "the fuck???"

**Screenshots**
later sorgy :(

**Additional context**
none

**Comment by muburu** [comment]
oh yeah i thought this was weird, was thinking they added this back to gibbing in general but apparently it's just vox?? weird..

**Comment by lzk228** [comment]
afaik there is an issue about all mobs dropping limbs on gibbing

**Comment by bruhmogus** [comment]
> afaik there is an issue about all mobs dropping limbs on gibbing

yes but it should probably be disabled until its actually useful so that its less confusing

**Comment by thebadman4662** [comment]
> afaik there is an issue about all mobs dropping limbs on gibbing

Other than vox I have not seen any other limbs in "forever"

**Comment by Golinth** [comment]
I kinda feel the opposite tbh, I miss the body parts after a gibbing

**Comment by thebadman4662** [comment]
> I kinda feel the opposite tbh, I miss the body parts after a gibbing

Extra blood to make Nec, Ocu and of course fats for soap. :godmode:

**Comment by saga3152** [comment]
> Actually, how do we even get fat now for soap reaciton? 🤔
Grind meat
